dali/base/dasds.cpp

```diff
@@ -1624,7 +1624,7 @@ class CXMLFileExternal : public CExternalFile, implements IExternalHandler
     virtual void resetAsExternal(IPropertyTree &_tree)
     {
         PTree &tree = *QUERYINTERFACE(&_tree, PTree);
-        tree.clear();
+        Owned<IPropertyTree> toDelete = tree.detach();
```

Comment:
Is this equivalent to ::Release(tree.detach());

And if so, would it not have been clearer to code it that way?